docs/api-reference/deck.md

```diff
@@ -297,6 +297,18 @@ deck.setProps({...});
 
 See the Properties section on this page for more detail on which props can be set.
 
+##### `redraw`
+
+Attempt to draw immediately, rather than waiting for the next draw cycle. By default, deck flushes all changes to the canvas on each animation frame. This behavior might cause the deck canvas to fall out of sync with other components if synchronous updates are required.
+
+Redrawing frequently outside of rAF may cause performance problems. Only use this method if the render cycle must be managed manually.
```

Comment:
`rAF` sounds like something that people may not understand on first read maybe type it out or add some link or extra paragraph?